toml/parse_test.ts

```diff
@@ -25,7 +25,7 @@ import {
 import { parse } from "./parse.ts";
 
 Deno.test({
-  name: "[TOML parser] Scanner",
+  name: "Scanner()",
```

Comment:
```suggestion
  name: "Scanner",
```
I discussed this Yoshiya. Let's omit the brackets from class names. It being in PascalCase should be enough to signify it's a class.